core/controllers/contributor_dashboard.py

```diff
@@ -1047,7 +1052,11 @@ class ContributorAllStatsSummariesHandler(
 
     @acl_decorators.can_fetch_all_contributor_dashboard_stats
     def get(self, username: str) -> None:
-        """Handles GET requests."""
+        """Fetches contributor dashboard statistics.
```

Comment:
```suggestion
        """Fetches stats for given contributor.
```

Reply:
I have updated the docstring with the suggested change.